tests/speed-comparison.el

```diff
@@ -0,0 +1,21 @@
+(with-current-buffer (find-file-noselect "/home/arne/tmp/edn2.list")
```

Comment:
Maybe include this in the repo, or make it a relative path to some place where devs can set up their own test files locally.